packages/docusaurus-utils/src/index.ts

```diff
@@ -118,3 +118,12 @@ export {
 export {isDraft, isUnlisted} from './contentVisibilityUtils';
 export {escapeRegexp} from './regExpUtils';
 export {askPreferredLanguage} from './cliUtils';
+
+export {
+  getFileLastUpdate,
+  type LastUpdateData,
+  type FrontMatterLastUpdate,
+  readLastUpdateData,
+  GIT_FALLBACK_LAST_UPDATE_AUTHOR,
+  GIT_FALLBACK_LAST_UPDATE_DATE,
```

Comment:
hmm that annoys me a bit to export those, but at the same time if we don't we'd have to erase the blog tests you wrote, so let's keep this for now 🤷‍♂️